src/enclave/enclave_init.c

```diff
@@ -166,9 +234,15 @@ int __libc_init_enclave(int argc, char** argv)
     SGXLKL_VERBOSE("calling _lthread_sched_init()\n");
     _lthread_sched_init(cfg->stacksize);
 
-    if (lthread_create(&lt, NULL, startmain, NULL) != 0)
+    /* Run startmain() in a new lthread */
     {
-        sgxlkl_fail("Failed to create lthread for startmain()\n");
+        static startmain_args_t args;
```

Comment:
For my understanding, is this(creating code blocks with parantheses) a standard C feature or a gcc extension?

Reply:
Also wondering if this was intentional, and if yes what was the rationale behind it.